### ISSUE TITLE
Install-WaykNow always tries to install x86 even tho os is x64

### ISSUE THREAD

**Issue by si458** [Devolutions/WaykNow-ps#27]
Install-WaykNow always tries to install x86 even though the windows 10 os is x64

[System.Environment]::Is64BitOperatingSystem = true
$Env:PROCESSOR_ARCHITECTURE -eq 'ARM64' = false
$Env:PROCESSOR_ARCHITEW6432 -eq 'ARM64' = false

**Comment by si458** [comment]
version 2020.1.5 works fine 
```
PS C:\Windows\system32> Install-Module -Name WaykNow -RequiredVersion 2020.1.5.0 -Force
PS C:\Windows\system32> Import-Module WaykNow
PS C:\Windows\system32> Get-WaykNowPackage
Url                                                                             Version
---                                                                             -------
https://cdn.devolutions.net/download/Wayk/2020.2.0.0/WaykNow-x64-2020.2.0.0.msi 2020.2.0
```

but version 2020.2.0 keeps returning the x86 version?
```
PS C:\Windows\system32> Install-Module -Name WaykNow -RequiredVersion 2020.2.0 -Force
PS C:\Windows\system32> Import-Module WaykNow
PS C:\Windows\system32> Get-WaykNowPackage
Url                                                                             Version
---                                                                             -------
https://cdn.devolutions.net/download/Wayk/2020.2.0.0/WaykNow-x86-2020.2.0.0.msi 2020.2.0
```

**Comment by thenextman** [comment]
Thanks for the bug report. This was a regression and is fixed in master. We'll publish a 2020.2.1 release on PSGallery ASAP incorporating the fix.

**Comment by thenextman** [comment]
Fix commit: 36d86675b05347de601c52c0d3fc735af89d9c27

**Comment by thenextman** [comment]
2020.2.1 now on PSGallery
